packages/swap/src/event-handlers/__tests__/swapRequested.test.ts

```diff
@@ -249,12 +292,35 @@ describe(swapRequested, () => {
   });
 
   it('creates a new swap request (VAULT 180)', async () => {
-    await swapRequested({ prisma, event: { ...event, args: vault180 }, block });
+    await swapRequested({ prisma, event: { ...event, args: vaultBitcoin180 }, block });
+
+    const request = await prisma.swapRequest.findFirstOrThrow();
+
+    expect(request).toMatchSnapshot({
+      id: expect.any(BigInt),
+    });
+  });
+
+  it('creates a new swap request and gets the solana tx ref (VAULT 180)', async () => {
+    env.SOLANA_RPC_HTTP_URL = 'https://rpc.devnet.solana.chainflip.io:443';
```

Comment:
this env is already specified in swap's vitest config, so there no need to set it here.

Reply:
nice - removed it